sources/packages/backend/apps/db-migrations/src/sql/Institution/Add-col-other-regulating-body.sql

```diff
@@ -0,0 +1,4 @@
+-- Add other_regulating_body column to institutions table
+ALTER TABLE sims.institutions ADD COLUMN IF NOT EXISTS other_regulating_body VARCHAR(100);
+
+COMMENT ON COLUMN sims.institutions.other_regulating_body IS 'Allows the user to enter the description if the option other is selected from the regulating body dropdown';
```

Comment:
Please add a period at the end of the sentence.

Reply:
Thanks. Will do.